### ISSUE TITLE
update dotnet SDK 2.2?

### ISSUE THREAD

**Issue by liguobao** [burakince/docker-dotnet-sonarscanner#4]

**Comment by kawazoe** [comment]
I agree, it would be nice to have a build of this container for each of the .net core releases. Right now people are forking this repo and publishing their own fork to docker hub. As far as I know, they are not maintained either.
